uglyui/src/services/Server.js

```diff
@@ -2,7 +2,8 @@ import axios from "axios";
 import { getInstance } from "../auth";
 
 const apiClient = axios.create({
-  baseURL: `http://localhost:8000`,
+  // baseURL: `http://localhost:8000`,
+  baseURL: `/api`,
```

Comment:
How does this work? Doesn't the server run on a different port than the client when you run locally?

Reply:
Good catch. Can we put that in a config file that I can replace when uploading to AWS?

Reply:
What I've done in the past is pretty ugly: as part of the production build process I run "sed" on a constants file. I'll see if webpack has improved that situation and let you know.